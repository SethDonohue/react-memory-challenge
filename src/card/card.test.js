import React from 'react';
import renderer from 'react-test-renderer';
// import { shallow } from 'enzyme';
import Card from './card';

// const Enzyme = require('enzyme');

// const EnzymeAdapter = require('enzyme-adapter-react-16');

// Enzyme.configure({ adapter: new EnzymeAdapter() });


// it('renders a card', ()=> {
//   const component = shallow(<Card />);
//   // const cardText = <p>10</p>;
//   expect(component.hasClass('card')).toEqual(true);
// });

test('renders a card', ()=> {
  const component = renderer.create(
    <Card />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});