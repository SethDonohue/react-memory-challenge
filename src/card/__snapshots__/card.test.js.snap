// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a card 1`] = `
<div
  className="card"
  id={undefined}
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
  type="button"
>
  <p
    style={
      Object {
        "display": undefined,
      }
    }
  />
</div>
`;
